pkgs/by-name/si/sillytavern/sillytavern.desktop

```
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Icon=sillytavern
+Name=SillyTavern
+Exec=sillytavern
+Terminal=true
+Hidden=false
+Categories=Education
```

Is this the better category for this package?

---

I would say go with `Network;Chat`, where chat is listed as an [Additional Category](https://specifications.freedesktop.org/menu-spec/latest/apas02.html) in the freedesktop spec.

---

For easy committing:

```suggestion
Categories=Network;Chat
```